data/en/directorycopy.json

```diff
@@ -18,6 +18,15 @@
 	},
 	"links": [
 
+	],
+	"examples": [
+		{
+			"title":"Simple DirectoryCopy Example",
+			"description":"Copy directory from one place to another.",
+			"code":"<cfoutput>#directoryCopy(expandPath(\"./mySourceDirectory\"),expandPath(\"../MyDestinationDirectory\"))#</cfoutput>",
```

Comment:
It is not necessary to add the `<cfoutput>` and `#` symbols around a single function call, one liner. It will be wrapped automatically, so to clarify you can just use this instead:

"code": "directoryCopy(expandPath(\"./mySourceDirectory\"),expandPath(\"../MyDestinationDirectory\"));"